commands/latexify.go

```diff
@@ -0,0 +1,32 @@
+package commands
+
+import (
+	"fmt"
+	"github.com/anhgelus/gokord/utils"
+	"github.com/bwmarrin/discordgo"
+	"github.com/nyttikord/gomath"
+)
+
+func Latexify(s *discordgo.Session, i *discordgo.InteractionCreate) {
+	resp := utils.NewResponseBuilder(s, i).IsEphemeral()
+
+	expr := i.Interaction.ApplicationCommandData().GetOption("expression").StringValue()
```

Comment:
Use `optMap := utils.GenerateOptionMap(i)` and check if the option is present like this:
```go
expr, ok := optMap["expression"]
if !ok {
    // not present
}
```